server/subscribe.go

```diff
@@ -587,7 +587,7 @@ func (p *Plugin) hasPermissionToManageSubscription(instanceID types.ID, userID,
 
 	switch cfg.RolesAllowedToEditJiraSubscriptions {
 	case "team_admin":
-		if !p.API.HasPermissionToChannel(userID, channelID, model.PERMISSION_MANAGE_TEAM) {
+		if !p.API.HasPermissionToChannel(userID, channelID, model.PermissionManageTeam) {
```

Comment:
I wonder if this should be `HasPermissionToTeam` or if this still works like this

Reply:
maybe @hanzei can comment this

Reply:
I don't think the semantics for permissions have changed. Also, the server does fall back to `HasPermissionToTeam`: https://github.com/mattermost/mattermost-server/blob/21639f9c8730b2f49ee52322acd3acdbf64ac6d9/app/authorization.go#L190